guides/request-validation-csharp-web-api/example-2/example-2.4.x.cs

```diff
@@ -0,0 +1,61 @@
+using System.Net.Http;
+using System.Text;
+using System.Web.Http;
+using Twilio.TwiML;
+using ValidateRequestExample.Filters;
+
+namespace ValidateRequestExample.Controllers
+{
+    public class TwilioMessagingRequest
+    {
+        public string Body { get; set; }
+    }
+
+    public class TwilioVoiceRequest
+    {
+        public string From { get; set; }
+    }
+
+    public class IncomingController : ApiController
+    {
+        [Route("voice")]
+        [AcceptVerbs("POST")]
+        [ValidateTwilioRequest]
+        public HttpResponseMessage PostVoice([FromBody] TwilioVoiceRequest voiceRequest)
+        {
+            var message =
+                "Thanks for calling! " +
+                $"Your phone number is {voiceRequest.From}. I got your call because of Twilio's webhook. " +
```

Comment:
@acamino can you split this line to make it more readable

```cs
$"Your phone number is {voiceRequest.From}. " +
"I got your call because of Twilio's webhook. " +
```

Reply:
Thanks for the suggestion @joliveros!